wrappers/matlab/option.m

```diff
@@ -90,6 +90,7 @@
         auto_gain_limit                 (86)
         auto_rx_sensitivity             (87)
         transmitter_frequency           (88)
-        count                           (89)
+        vertical_binning                (89)
```

Comment:
How is vertical binning 88 elsewhere but 89 here? There's a bug somewhere...

Reply:
fixed on the other place .